qa/L0_backend_python/test.sh

```diff
@@ -414,7 +414,7 @@ if [ "$TEST_JETSON" == "0" ]; then
     # [DLIS-5970] Disable io tests for Windows since GPU Tensors are not supported
     # [DLIS-6122] Disable model_control & request_rescheduling tests for Windows since they require load/unload
     if [[ ${TEST_WINDOWS} == 0 ]]; then
-        SUBTESTS+=" variants io python_based_backends async_execute"
+        SUBTESTS+=" variants io python_based_backends async_execute response_sender"
```

Comment:
Is there anything specific about this test that won't be supported on windows?

Reply:
I don't see it now. I think we can move it up and if there is a failure then move it back here?